docs/reference/snapshot-restore/apis/get-snapshot-api.asciidoc

```diff
@@ -115,6 +115,15 @@ List of <<data-streams,data streams>> included in the snapshot.
 (Boolean)
 Indicates whether the current cluster state is included in the snapshot.
 
+[[get-snapshot-api-feature-states]]
+`feature_states`::
+(array)
+List of feature states which were included when the snapshot was taken,
+including the list of system indices included as part of the feature state. The
+`feature_name` field of each can be used in the `feature_states` parameter when
+restoring the snapshot to restore a subset of feature states. Only present if
```

Comment:
Is the part about only being present if one or more feature states are included? In the example below, an empty array is shown.